{{cookiecutter.project_root_dir_name}}/circle.yml

```diff
@@ -0,0 +1,53 @@
+# You need to setup the DOCKER_USER, DOCKER_EMAIL, and DOCKER_PASS
```

Comment:
They can use whatever, it seemed nice to provide a boilerplate for the CI/CD too. It is not a requirement, it is more an added bonus.